be/src/runtime/query_statistics.cpp

```diff
@@ -33,7 +33,7 @@ void QueryStatisticsRecvr::insert(const PQueryStatistics& statistics, int sender
     } else {
         query_statistics = iter->second;
     }
-    query_statistics->merge_pb(statistics);
+    query_statistics->from_pb(statistics);
```

Comment:
Shouldn't it be accumulation here?

Reply:
The query statistics such as scan_rows refers to ```RuntimeProfile::Counter _raw_rows_counter``` have been already accumlated in counter and the values in query statistics are not delta.
The query statistics sent by different senders should be merged in Exchange Nodes. 
However, the query statistics sent by the same sender why should we merge them here.
If merged, the query statistics will be enlarged by repeat accumulation when sql with limit.

Reply:
I see, I don’t think it’s a problem